includes/actions/class-amp-frontend-actions.php

```diff
@@ -0,0 +1,19 @@
+<?php
+// Callbacks for adding AMP-related things to the main theme
+
+require_once ( AMP__DIR__ . '/includes/actions/class-amp-actions.php' );
+
+class AMP_Frontend_Actions extends AMP_Actions {
```

Comment:
I'm not sure if this class needs to extend `AMP_Actions` since it won't use any of the abstract class' methods.